pkg/kine/server/otel.go

```diff
@@ -0,0 +1,30 @@
+package server
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/metric"
+)
+
+const name = "k8s-dqlite"
```

Comment:
can we have a more descriptive name? While not exported, this constant will be available in the whole package and will give very little information to whoever sees it (like in auto-complete)